test/system/250-systemd.bats

```diff
@@ -125,4 +125,33 @@ function service_cleanup() {
     service_cleanup
 }
 
+@test "podman generate - systemd template" {
+    cname=$(random_string)
+    run_podman run -dt --name $cname $IMAGE top
+
+    run_podman generate systemd --new --template -n $cname
+    is "$output" ".*%I.*" "%I indentifiers in template"
+    is "$output" ".*%i.*" "%i indentifiers in template"
+}
+
+@test "podman generate - systemd template no support for pod" {
+    cname=$(random_string)
+    podname=$(random_string)
+    run_podman pod create --name $podname
+    run_podman run --pod $podname -dt --name $cname $IMAGE top
+
+    run_podman 125 generate systemd --new --template -n $podname
+    is "$output" ".*--template is not supported for pods.*" "Error message contains 'not supported'"
+
+    run_podman rm -f $cname
+    run_podman pod rm -f $podname
+}
+
+@test "podman generate - systemd template only used on --new" {
+    cname=$(random_string)
+    run_podman create --name $cname $IMAGE top
+    run_podman 125 generate systemd --template -n $cname
+    is "$output" ".*--template requires --new.*" "Error message should be '--template requires --new'"
+}
```

Comment:
Currently, we generate the templates only but do not run them.  To catch regressions and make sure that everything's working, we really need to run them via `systemctl start ...` as done in the other tests.